### PR TITLE
New UI for chat input

### PR DESCRIPTION
**Pull request by Ferossgp**

### CODE REVIEW

**Comment by errorists** [comment]
inb4, this PR is for the redesign of the chat input. I was too lazy to create an issue first but it's been heavily discussed between me and Gheorghe. This work is awesome! 
[Designs in Figma](https://www.figma.com/file/aS1ct66VQ6V0cio7vSqS8UoG/Chat?node-id=7782%3A36147)

**Comment by oskarth** [comment]
Small request - could you please rebase to latest master so this commit https://github.com/status-im/status-react/commit/b5fda1234e5447cbc626059dfc78fcfbabc9e926 gets included? This means we aren't running E2E tests on eth.prod cluster anymore, and since this happens on every push, the impact is quite big. This ensures we have more accurate metrics going forward from Jul 1 onward, which would be awesome. See https://discuss.status.im/t/user-growth-and-retention/1782 for more

**Comment by hesterbruikman** [comment]
Looks great! Tried on

Samsung Galaxy S10e - Android 10
iPhone 8 - iOS 13
Fairphone 2 - Android 7

Couldn't figure out how to `send image`. No `send arrow` on right side of input field. I'm not 100% sure on how I'd expect this to behave (cc @errorists ). Probably sticker on the right side of the input field should switch to `send arrow`. Can't seem to add a sticker anyway

Screenshot_20200713-121723_Status PR

**Comment by errorists** [comment]
correct, I'm guessing that's a bug
Screenshot 2020-07-13 at 12 35 18

**Comment by churik** [comment]
@Ferossgp several issues:

### ISSUE 1: Text is too dark in dark mode (IOS)
IMAGE 2020-07-14 12:21:47

### ISSUE 2: Autocorrection panel hint is overlapped input (IOS)
All fine on current nightly.
**Steps:**
1. Type some non-existing word in input 
2. Go back to home view and open corresponding chat again
3. Tap on word
IMAGE 2020-07-14 12:25:24

### ISSUE 3: Jumping keyboard on navigation between chat view <-> home view in case when focus is set in input field (IOS)
ezgif com-video-to-gif (3)
Also for me it is more noticeable (in 1-1 chat) lagging animation (navigate to 1-1 chat -> big chat icon (start of chat) is shown for a few moments) comparing to current nightly.

### ISSUE 5:  Close icon is not visible on Dark mode when sending image (IOS, Android)
IMAGE 2020-07-14 12:47:01

**Comment by churik** [comment]
### ISSUE 6: Missing accessibility-ids for elements in chat input
1) 'show-extensions-icon' for Commands
2) 'show-stickers-icon' for stickers
3) 'show-photo-icon' for images

**Review by cammellos — APPROVED** [review]

**Comment by churik** [comment]
@Ferossgp 
### ISSUE 7: bottom sheet is not closed when navigating to + > Start new chat (Android only, not reproducible on nightly)
ezgif com-video-to-gif (5)

**Comment by churik** [comment]
@Ferossgp 
one more accessibility-id: 'cancel-message-reply' to "Cancel" icon in Reply.

File: src/status_im/ui/screens/chat/image/views.cljs [hunk 1]
```diff
@@ -30,7 +30,8 @@
      [react/image {:style  (merge {:width            wh
                                    :height           wh
                                    :background-color :black
-                                   :border-radius    4}
+                                   :border-radius    4
+                                   :resize-mode      :center}
```
Comment (Ferossgp) [inline src/status_im/ui/screens/chat/image/views.cljs]: To remove in this PR

**Comment by churik** [comment]
@Ferossgp 
nice work!
Tested on IOS 13 and Android 10.
Please mind my commits while squashing.

**Review by churik — APPROVED** [review]

**Pull request merged**